go.mod

```diff
@@ -33,22 +36,27 @@ require (
 	github.com/imdario/mergo v0.3.6 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/jsonnet-bundler/jsonnet-bundler v0.5.1 // indirect
```

Comment:
This was probably not intended. :)

PS. I believe we can drop the `go.mod` update entirely.

Reply:
done